components/vars/css/components/spectrum-dialog.css

```diff
@@ -4,7 +4,7 @@
   --spectrum-dialog-confirm-icon-color: var(--spectrum-global-color-gray-900);
   --spectrum-dialog-confirm-title-text-color: var(--spectrum-global-color-gray-900);
   --spectrum-dialog-confirm-description-text-color: var(--spectrum-global-color-gray-800);
-  --spectrum-dialog-confirm-width: 0.9;
+  --spectrum-dialog-confirm-width: 90%;
```

Comment:
Note that these variables were not used, so no error was seen. Only the avatar percentage variable was used, hence it being the only error we're fixing here.